radio/src/datastructs.h

```diff
@@ -854,6 +868,15 @@ PACK(struct RadioData {
   char ownerRegistrationID[PXX2_LEN_REGISTRATION_ID];
 
   GYRO_FIELDS
+//OW
+#if defined(TELEMETRY_MAVLINK)
+  uint16_t mavlinkBaudrate:3;
+  uint16_t mavlinkBaudrate2:3;
+  uint16_t mavlinkExternal:2;
+  uint16_t _mavlinkDummy:8; // currently not used
```

Comment:
If that one is not used, is there a reason for using `uint16_t` rather than `uint8_t`?